QSB/TransformSync/PlayerCameraSync.cs

```diff
@@ -0,0 +1,51 @@
+﻿using OWML.ModHelper.Events;
+using QSB.Animation;
+using QSB.Events;
+using QSB.Utility;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace QSB.TransformSync
+{
+    public class PlayerCameraSync : TransformSync
+    {
+        public static PlayerCameraSync LocalInstance { get; private set; }
+
+        public override void OnStartLocalPlayer()
+        {
+            LocalInstance = this;
+        }
+
+        uint GetAttachedNetId()
+        {
+            return netId.Value - 2; // This is the 3rd transformsync in the "stack"
```

Comment:
Interestiiing. Probably a good idea to leave a comment here explaining why this is needed to be done at all, since it is quite peculiar. Comments are more useful when they answer "why" instead of "what".

Plus I'd like to understand it out of curiosity.

Reply:
`GetComponent<TheMainNetworkThing>().netId.Value` instead?

Reply:
the camera and player are different gameobjects, same with the ship

Reply:
We're considering the netId of the *main network thing* of a player to the the *player ID*, right? If so, maybe PlayerRegistry could keep track of those, so we could just say PlayerRegistry.MyId or similar?

Reply:
Yeah, that doesn't solve this issue...
Anyway, the current solution seems fragile, will break if component order changes, right?